adhocracy4/actions/models.py

```diff
@@ -46,12 +46,19 @@ def exclude_updates(self):
 
 
 class Action(models.Model):
-
     # actor, if actor is None the action was create by the system
     actor = models.ForeignKey(
         settings.AUTH_USER_MODEL, on_delete=models.CASCADE, blank=True, null=True
     )
 
+    target_creator = models.ForeignKey(
+        settings.AUTH_USER_MODEL,
+        related_name="target_creator",
+        on_delete=models.CASCADE,
```

Comment:
do we want to cascade here? I guess so?

Reply:
On the other hand, when we don't do that for some reason, the action should also stay.
I mean we have: action - target - target creator
Normally when the target creator is deleted that cascades though the target to the action. But if for some reason, we do not delete the target when the creator is deleted, we also shouldn't delete the action. So I would say we want SET_NULL
Even though, we do the same for the actor, but that makes more sense as a n action always has an object and an actor, but not always a target. Right?